### ISSUE TITLE
Nikon D780 Tethering does not work

### ISSUE THREAD

**Issue by duelle** [darktable-org/darktable#9581]
Even though the troubleshooting steps (using `gphoto2`) seem to work perfectly fine, it is not possible to use tethering with `darktable`. The option to use tethering is not available in the _import_ module and selecting the tethering view results in the error message `no camera with tethering support available for use...`.

**To Reproduce**
1. Connect Nikon D780 with PC via USB cable 
2. Turn on Nikon D780
3. Run `darktable`
4. Check `import` module (only `mount camera` option available)
5. Open `tethering` view (see message: _no camera with tethering support available for use..._)

**Expected behavior**
- Allow tethering with Nikon D780 as it seems to be supported by `gphoto2`
- See `switch to tethered shooting` option in `import` module
- Availability of actual tethererd shooting in the `tethering` view

**Platform**
* darktable version : 3.6.0
* OS                : Linux Kernel 5.12.15-arch1-1
* Linux - Distro    : ArchLinux
* Memory            : 16 GB
* Graphics card     : NVidia GeForce RTX 2060
* Graphics driver   : 
* OpenCL installed  : yes
* OpenCL activated  :  yes
* Xorg              : 1.20.12-1
* Desktop           : i3wm

**Additional context**
- I was able to *successfully* use tethering via `gphoto2` in the troubleshooting process as provided here: https://www.darktable.org/usermanual/en/tethering/troubleshooting/
- Nonetheless, tethering with `darktable` seems not to work.

**Further information**
As instructed in the troubleshooting section for tethering, I logged the output of `darktable -d camctl 2>1 >camctl.log` which is the following:
```
% cat camctl.log 
0.803433 [camera_control] creating new context 0x5636e41833a0
0.823962 [camera_control] loaded 2562 camera drivers.
1.344988 [camera_control] loaded 38 port drivers.
1.358053 [camera_control] 1 cameras connected
1.358062 [camera_control] found new Nikon DSC D780 on port usb:001,018
18.097786 [camera_control] destroy darktable camcontrol
```

**Remarks**
In case there is any further information missing that could be of help for resolving this issue just let me know. I'll happily try to support to fix this to the best of my knowledge and abilities.

**Comment by ptilopteri** [comment]
your camera is not yet supported.  see:
https://www.darktable.org/resources/camera-support/
and
https://discuss.pixls.us/t/raw-samples-wanted/5420?u=lebedevri

**Comment by kmilos** [comment]
@ptilopteri D780 is supported at the most basic level, otherwise it wouldn't be in the list at all.

**Comment by duelle** [comment]
Thanks for the feedback.
I read the text of the tethering troubleshooting section that way, that in case it works properly with `gphoto2` it would be expected to also work in Darktable. And that one is supposed to open a ticket if that is not the case.

Anyways, an upload of raw samples to https://raw.pixls.us/ is on the way, containing samples for Nikon D780 with all variants of:
* Image size (S, M, L)
* RAW bit sizes (12bit, 14bit)
* Compression types (normal, losless)

If anything else is needed to support this case, please let me know.

**Comment by jenshannoschwalm** [comment]
Your nikon is supported in libgphoto2 since v 2.5.26, i guess you have at least this installed?

Uploading the files is not the point here as you want tethering, uploading is necessary if dt can't handle the files. I guess that is working...

I am asking because i did most of recent dev work on gphoto2 device handling in dt.

A first question after reading the log: after connection the camera and switching it on - did you ever see a button (in the import & tethering upper left corner) naming the camera offering you to mount it? Please thy again if you are not sure. In any case - if you want to tether or import files from the camera - it must be "mounted", dt does **not** do this automatically. From the logfile it seems you didn't do so, either because you didn't know to do so or because the button was not there. Please let me know ... (I might have further questions btw willing to track this down :-)

**Comment by duelle** [comment]
Thanks for you feedback @jenshannoschwalm 
Apparently, I seem to have misread the documentation of darktable. In one paragraph it said that one should not mount it if one wants to use tethering. When rereading it, that mounting referred to mounting the SD card in the OS and not the mounting inside darktable. After having learnt that, I could get darktable to use the camera in tethering mode (after using the mount option in the import module). Some options in the tethering view took a moment to respond, but apart from that, it seemed to work fine.

Regarding the upload of new samples, I saw that color card pictures were appreciated and the existing ones seemed not to be such. So I made some of that kind and uploaded them. I hope the samples are of any help.
